src/Microsoft.IdentityModel.Clients.ActiveDirectory/Platforms/iOS/iOSTokenCacheAccessor.cs

```diff
@@ -36,9 +36,24 @@
 
 namespace Microsoft.Identity.Core
 {
-    internal class iOSTokenCacheAccessor : ITokenCacheAccessor
+    internal class iOSTokenCacheAccessor : ITokenCacheAccessor, ILegacyCachePersistence
```

Comment:
This is for backwards compat with ADAL v3. I would rather not do this unless there are customers who cannot use MSAL directly. ADAL v3 -> ADAL v5 / MSAL never worked as far as I can tell.